tests/unit_tests/test_wave_06.py

```diff
@@ -109,16 +112,23 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That result is truthy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories, and that the items that were swapped are not there
-
-@pytest.mark.skip
+    assert result == True
+    assert len(tai.inventory) == 3 and len(jesse.inventory) ==3
+    for item in [item_a, item_b, item_f]:
+        assert item in tai.inventory
+    for item in [item_c, item_e, item_d]:
+        assert item in jesse.inventory
```

Comment:
👍 

Ooh! Yes. I love the `in` checks here, and nice use of the loops to minimize repetition!